docs/content_management/data_migration/importing_data.md

```diff
@@ -209,6 +209,27 @@ and then use it when creating the article.
 [[= include_file('code_samples/data_migration/examples/create_parent_and_child_content.yaml') =]]
 ```
 
+### Images
+
+The following example shows how to migrate
+a `public/var/site/storage/images/3/8/3/0/383-1-eng-GB/example-image.png` image between two instances.
+
+To prevent the manual addition of images to specific DFS or local locations, such as `public/var/site/storage/images/` you can move image files to, for example `src/Migrations/images`.
+Adjust the migration file and configure the `image` field data as follows:
+
+```yaml
+        -   fieldDefIdentifier: image
+            languageCode: eng-GB
+            value:
+                alternativeText: ''
+                fileName: example-image.png
+                path: src/Migrations/images/example-image.png
+```
+
+This migration copies the image to the appropriate directory, 
+in this case `public/var/site/storage/images/3/8/3/0/254-1-eng-GB/example-image.png`,
+enabling swift file migration between instances regardless of storage (local, DFS).
```

Comment:
> file migration between instances

Is that really the main purpose? My intent of using such a migration is to create content during deployment automatically (because the content are related to a specific new feature and has specific remote ids, for example) and as the content has a required image field, it must be provided in the migration.